components/server/src/ome/security/SystemTypes.java

```diff
@@ -78,6 +79,8 @@ public boolean isSystemType(Class<?> klass) {
             return true;
         } else if (IGlobal.class.isAssignableFrom(klass)) {
             return true;
+        } else if (Namespace.class.isAssignableFrom(klass)) {
```

Comment:
We do. While, say, `count_Image_annotationLinks_by_owner` is a view, `count_Experimenter_annotationLinks_by_owner` is a table. Is that okay: should I go ahead and make `Namespace` global and have a `count_Namespace_annotationLinks_by_owner` table?